### ISSUE TITLE
incompatibity

### ISSUE THREAD

**Issue by GMAkroma** [mezz/JustEnoughItems#776]
![wwww](https://cloud.githubusercontent.com/assets/26563833/24134372/137ea8c2-0dc1-11e7-80b1-0a1414d0399a.png)
Thank you for reporting!

For issues:
 * If the issue seems to affect only one mod, report to them instead of here.
 * Remove NEI and FEI to see if they are causing the issue before reporting.
 * If you crashed, please paste the crash log to [gist](https://gist.github.com/) and link it here.

For suggestions:
 * Please do not suggest features from NEI. [See this spreadsheet](https://goo.gl/MQ0f3R) before requesting.

**Comment by GMAkroma** [comment]
IS NOT WORKING under java 64 BITS

**Comment by TehNut** [comment]
Update Forge.

**Comment by BrendanatorX** [comment]
I'm having issues with Just Enough Items (JEI) not working either.
I've tried many different versions for mc 1.10.2 to get it working in my personal modpack but upon trying to load any world, it just says "Loading World" forever..

Upon testing, it seems my issue is about incompatibility but it is very complicated.

I have discovered so far that the mods
'Armory - Armoring the World'
'Armory - Weaponry'
'Enchanting Plus'
'Ender I/O'
'Just Enough Items'

These are what I have found so far in this web of incompatibility. disabling one of these mods seems makes the rest work fine together in my personal modpack, but all together with no other mods in the mix seems to still work. I will by process of elimination work out a few other mods and list them here.
